docs/milestones.md

```diff
@@ -0,0 +1,13 @@
+# Milestones
+
+[M0] Estructura de datos que permita la gestión de vehículos, materia prima, almacén y pedidos
```

Comment:
Una estructura de datos no es un producto. Por favor, no repitáis los errores del año pasado. Mira los objetivos de la asignatura para ver exactamente qué es lo que habría que entregar.